addons/website/__manifest__.py

```diff
@@ -101,4 +100,334 @@
     'application': True,
     'post_init_hook': 'post_init_hook',
     'uninstall_hook': 'uninstall_hook',
+    'assets': {
+        'assets_snippet_s_media_list_css_000': [
```

Comment:
Correct me if I'm wrong but you take the template `id` to generate bundles? In this case (and actually most of them if not all) you should take the `inherit_id` (if any) since the `id` is only used for identification.

Here the template inherits from `web.assets_frontend` and should then be appended inside of it instead of creating a new bundle (that will never be called).